# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-disable-unknown-compiler-warning.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-133.0.6943.142/build/config/compiler/BUILD.gn.vanilla	2025-03-03 18:09:26.306100726 +0100
+++ chromium-133.0.6943.142/build/config/compiler/BUILD.gn	2025-03-03 18:09:56.718321304 +0100
@@ -1992,7 +1992,7 @@
         cflags_cc += [ "-Wno-dangling-assignment-gsl" ]
 
         # TODO(crbug.com/376641662): Fix and re-enable.
-        cflags += [ "-Wno-nontrivial-memcall" ]
+        #cflags += [ "-Wno-nontrivial-memcall" ]
       }
     }
 
